source/ceylon/test/engine/internal/file.ceylon

```diff
@@ -0,0 +1,39 @@
+import java.io {
+    File,
+    FileOutputStream,
+    PrintWriter
+}
+
+native class FileFacade(String path) satisfies Destroyable {
```

Comment:
Is there a reason why it’s an interface and a create function instead of just a native class?